src/Compute/Compute.Test/Compute.Test.csproj

```diff
@@ -12,7 +12,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Graph.RBAC" Version="3.4.0-preview" />
-    <PackageReference Include="Microsoft.Azure.Management.Compute" Version="43.0.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Compute" Version="43.0.0-preview.1" />
```

Comment:
Several test projects reply on 43.0.0-preview. You should update all of them, or assembly loading error will be reported.
Since you are upgrade the management sdk to 44.0, you can fix the issues in this PR after you finish upgrading to 44.0.
If you don't want this PR bloked, you can use 43.0.0-preview.1 for alll the test projects that reply on Microsoft.Azure.Management.Compute.

Reply:
@msJinLei, I think I updated them all, I will investigate the remaining build errors.